mandatory/4-tax.js

```diff
@@ -5,7 +5,13 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+return price * 1.2;
+}
```

Comment:
That is correct.

You could also do something lie `const amountOfTax = originalPrice * (taxRate / 100);` to calculate the tax and then add that tax to the original price, but your solution is also correct.